scripts/tests.sh

```diff
@@ -54,7 +48,7 @@ $ETH_PATH --test -d /tmp/test &
 # is available and is ready for the unit-tests to start talking to it.
 while [ ! -S /tmp/test/geth.ipc ]; do sleep 2; done
 echo "--> IPC available."
-
+sleep 2
```

Comment:
No, checking whether the file is available helps, that is done in line 59. What I was asking about was: Did you experience things like race conditions here that forced you to add the two second delay?